fe/model_utils.py

```diff
@@ -4,7 +4,20 @@
 
 from simtk.openmm import app
 from rdkit import Chem
-from rdkit.Geometry import Point3D
+
+
+def assert_mol_has_all_hydrogens(mol: Chem.Mol):
+    atoms = mol.GetNumAtoms()
+    mol_copy = Chem.AddHs(mol)
+    assert atoms == mol_copy.GetNumAtoms(), "Hydrogens missing for mol"
+
+
+def verify_rabfe_pair(mol: Chem.Mol, blocker: Chem.Mol):
+    assert_mol_has_all_hydrogens(mol)
+    assert_mol_has_all_hydrogens(blocker)
+    mol_charge = Chem.GetFormalCharge(mol)
+    blocker_charge = Chem.GetFormalCharge(blocker)
+    assert mol_charge == blocker_charge, f"Formal charge disagrees: blocker: {blocker_charge:d} ligand: {mol_charge:d}"
```

Comment:
Might comment if there are conditions we should make this line more or less stringent.

(Most stringent: `mol_charge == 0, blocker_charge == 0`, less stringent: `mol_charge == blocker_charge`, `abs(mol_charge - blocker_charge) < 2` etc.)

Reply:
Based on discussions, I thought we wanted to avoid any 'charge hopping'.